interface/locales/de/common.json

```
@@ -217,47 +217,47 @@
   "extensions": "Erweiterungen",
   "extensions_description": "Installiere Erweiterungen, um die Funktionalität dieses Clients zu erweitern.",
   "fahrenheit": "Fahrenheit",
-  "failed_to_add_location": "Failed to add location",
+  "failed_to_add_location": "Fehler beim Hinzufügen der Location",
```

```suggestion
  "failed_to_add_location": "Fehler beim Hinzufügen des Standorts",
```

Keeping it consistent